people/roger-chen.md

```diff
@@ -0,0 +1,6 @@
+Roger Chen
+-------------
+
+![](photos/roger.jpg)
```

Comment:
Please rename `roger.jpg` to `roger-chen.jpg`, there're other Roger's I am sure ;)